src/CMakeLists.txt

```diff
@@ -62,8 +62,8 @@ add_library(REGothEngine STATIC
   components/Sky.hpp
   components/Sky.cpp
   core.hpp
-  core/AbstractEngine.hpp
-  core/AbstractEngine.cpp
+  core/EmptyEngine.hpp
+  core/EmptyEngine.cpp
```

Comment:
I think `EmptyGame` would be more appropriate, that was my initial suggestion too

Reply:
Fixed now with 6679d33. Excuse me, I wasn't paying attention yesterday :(